subgraphs/enzyme-core/mappings/v4/NoDepegOnRedeemSharesForSpecificAssetsPolicy.ts

```diff
@@ -0,0 +1,9 @@
+import { ensureNoDepegOnRedeemSharesForSpecificAssetsPolicy } from '../../entities/NoDepegOnRedeemSharesForSpecificAssetsPolicy';
+import { FundSettingsUpdated } from '../../generated/contracts/NoDepegOnRedeemSharesForSpecificAssetsPolicy4Events';
+
+export function handleFundSettingsSet(event: FundSettingsUpdated): void {
+  let policy = ensureNoDepegOnRedeemSharesForSpecificAssetsPolicy(event.params.comptrollerProxy, event.address, event);
+  policy.assetConfigs = event.params.assetConfigs;
```

Comment:
This won't work. You will need to decode `event.params.assetConfigs` and then create the `AssetConfig` entities.

Reply:
done